mantidproject/mantid
# Indirect Data Analysis - FABADA PDF plotting options

**Description of work.**
This PR introduces an MVP style for the plotting options at the bottom of the MSDFit, IqtFit, ConvFit and JumpFit interfaces.
This PR introduces plotting options for the FABADA PDF output workspace in ConvFit and IqtFit.

**To test:**
1.`Interfaces`->`Indirect`->`Data Analysis`->`IqtFit` interface
2. Load the data below
3. Set as `2` exponentials
4. Move the endx slider on the miniplot a little to the left
5. Select `FABADA` as the minimizer and ensure PDF is ticked
6. Set the Fit string to be `0-2`
7. Click run and wait
8. Look at the output plot options. Make sure that:
  - The plot options are disabled while plotting is taking place
  - The plot options will plot the relevant parameters (in the combobox) when `Plot` is clicked
  - Select the PDF Group. Select one of its workspaces and a parameter and make sure `Plot` works
9. Change minimizer and click run again. The output options should only have the parameter combobox and Plot button

Repeat for `ConvFit` but select One Lorentzian, Delta function and flat background. This will take much longer to run.

**Data Files**
**IqtFit:**
[irs26176_graphite002_iqt.zip](https://github.com/mantidproject/mantid/files/2782977/irs26176_graphite002_iqt.zip)
**ConvFit:**
[irs26176_withhistory_red.zip](https://github.com/mantidproject/mantid/files/2782978/irs26176_withhistory_red.zip)
[irs26173_graphite002_res.zip](https://github.com/mantidproject/mantid/files/2782979/irs26173_graphite002_res.zip)

Make sure the new unit test files pass.

Fixes #24534

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/IndividualTicketTesting/)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
